core/trigFilters.fcl

```diff
@@ -1280,10 +1280,105 @@ TrigAprFilters:{
     timeClusterCollection: "TTTZClusterFinder"
   }
 }
+################################################################################
+# Filters for sequences using RobustMultiHelixFinder
+#
+#
+################################################################################
+TrigMprFilters : {
+  mprDeHighPStopTargHSFilter: {
+    module_type: "HelixFilter"
+    helixSeedCollection: "TTMprHelixMergerDe"
+    posHelicitySelection : {
+      configured: true
+      doHelicityCheck: false
```

Comment:
this should be set to true because you define both helicity selections

Reply:
OK